<?xml version="1.0"?>
<launch>
  <arg name="bagfile_path" default="/home/ubuntu/Workspace/KIT/slamdog/bagfiles/8_good.bag"/>
  <arg name="image_topic" default="/camera/color/image_raw"/>

  <param name="use_sim_time" value="true"/>
  <param name="use_degree" value="true"/>


  <node pkg="rosbag" type="play" name="player" args="--clock $(arg bagfile_path)"></node>

  <include file="$(find my_aruco)/launch/groundtruth_publisher.launch" />

  <node pkg="my_aruco" type="my_aruco_node" name="my_aruco_node" output="screen" required="true">
    <param name="config_file" value="$(find my_aruco)/config/framos.yaml"/>
    <param name="image_topic" value="$(arg image_topic)"/>
  </node>


</launch>